File: Java/Sum_natural_numbers.java
package com.lev;

import java.util.Scanner;

public class Sum_natural_numbers {
    public static int natural(int n){
        n = (n * (n + 1)/2);
        return n;
    }
    public static void main(String[] args) {
        System.out.print("Enter n : ");
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        System.out.println("Sum of first "+ n +" natural numbers : "+natural(n));
    }
}

File: C++/Sum_natural_numbers.cpp
#include<iostream>
using namespace std;
int sumNatural(int n){
    n=(n*(n+1))/2;
    return n;
}
int main(){
    int n;
    cout<<"Enter n : ";
    cin>>n;
    cout<<"Sum of first "<<n<<" natural numbers : "<<sumNatural(n);
    return 0;
}
